.github/ISSUE_TEMPLATE/release-todo-issue-template.yml

```
@@ -0,0 +1,133 @@
+name: 🚀 Release To Do List
+description: Used to perform a release (Project maintainers only).
+title: "🚀Release To <TODO: release type here> - <TODO: release version number here>"
+body:
+- type: textarea
+  attributes:
+    label: Issues Released
+    placeholder: |
+      Create a list of all the items being released.
+      Example:
+      - #10
+      - #20
+    description: The list of issues that will be released.
+  validations:
+    required: true
+    render: markdown
+
+
+- type: checkboxes
+  attributes:
+    label: Perform Release ToDo List
+    description: Complete the following items to perform a release.
+    options:
+    - label: "Title updated by replacing the '**_\<TODO: release type here\>_**' section with the type of release."
+    - label: "Title updated by replacing the '**_\<TODO: release version number here\>_**' section with the version number."
```

Please remove the backslash symbols `\` from these 2 lines.

```suggestion
    - label: "Title updated by replacing the '**_<TODO: release type here>_**' section with the type of release."
    - label: "Title updated by replacing the '**_<TODO: release version number here>_**' section with the version number."
```